js/components/controlButton.js

```diff
@@ -29,27 +28,36 @@ class NonAccessibleButton extends React.Component {
       </a>
     );
   };
-};
+}
```

Comment:
Shouldn't we keep that semi colon?

Reply:
Not in this case, class declarations don't end with semicolons